src/index.tsx

```diff
@@ -117,7 +172,10 @@ const initialState = {
     inDevMode,
     searchBarValue: '',
     searchBarSuggestions: [],
-    shouldShowSearch: true
+    shouldShowSearch: true,
+    newOrgTitle: '',
```

Comment:
Not yet getting why these are here maybe cuz u want to track what’s being entered prior to the new organization being made via form submission. When does the new org go into the organizations list tho?